docs/erp_versionsuebergang.adoc

```diff
@@ -0,0 +1,608 @@
+= E-Rezept Antworten des Fachdienstes im Übergangszeitraum 2023 image:gematik_logo.png[width=150, float="right"]
+// asciidoc settings for DE (German)
+// ==================================
+:imagesdir: ../images
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:toc: macro
+:toclevels: 3
+:toc-title: Inhaltsverzeichnis
+
+IMPORTANT: DRAFT - Work in Progress!
+
+Auf dieser Seite sind relevante Informationen zum Verhalten des Fachdienstes während und nach der Übergangszeit aufgeführt. Dieser Übergangszeitraum wird nach der Technischen Anlage der KBV 6 Monate betragen. Gilt also somit vom 01.07.2023 bis 31.12.2023.
+
+toc::[]
+
+== Zustände des Fachdienstes im Übergangszeitraum
+
+Diese Darstellung zeigt die Konfigurationen der Umgebungen des Fachdienstes zu gegebener Zeit auf. Es ist dargestellt, welche Profilversionen vom Fachdienst akzeptiert werden.
+Folgende Konfigurationen wurden definiert:
+
+[cols="h,a"]
+[%autowidth]
+|===
+|Name | Eigenschaften
+
+| A | Konfiguration 2022, es werden nur bis 30.06.2023 gültigen Profile akzeptiert
+| B | Konfiguration FHIR 2022 + 2023, der Fachdienst akzeptiert die aktuell gültigen und die ab 01.07.2023 in Kraft tretenden Profile
+| C | Konfiguration 2023, der Fachdienst akzeptiert nur die ab 01.07.2023 in Kraft tretenden Profile
```

Comment:
die beschreibung passt nicht, denn in der tabelle unten ist zu sehen, dass alte profile länger auftreten

Reply:
besser: 
vor Übergangszeit 
während Übergangszeit
nach Übergangszeit

Reply:
Übergangszeit --> Übergangszeitraum